### ISSUE TITLE
A11y_Chat window_ScreenReader:Screen reader is reading message as sending whereas it is being displayed as send failed when using arrow keys to navigate.

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4560]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader users will get wrong information while navigating in scan mode.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Live region

### What is the public URL for the website?

https://web.powerva.microsoft.com/environments/839eace6-59ab-4243-97ec-a5b8fcc104e4/bots/107725f1-d827-41a4-8375-c91d1a97affc

### How to reproduce the issue?

1. Launch the application using URL: [Home - Test | Power Virtual Agents (microsoft.com)](https://web.powerva.microsoft.com/environments/839eace6-59ab-4243-97ec-a5b8fcc104e4/bots/107725f1-d827-41a4-8375-c91d1a97affc)
1. Tab till chat box and press enter to open it.
1. Start Narrator.
1. Type and send any message such that it fails to send.
1. Check is narrator is announcing information correctly.

### What do you expect?

Screen reader should read message correctly as displayed on screen.

### What actually happened?

Screen reader is reading message as sending whereas it is being displayed as send failed when using arrow keys to navigate.

### MAS reference

https://aka.ms/MAS1.3.1

### Additional context

Need investigation.

When in browse mode and screen reader is reading the whole page, for messages that has "Send failed. Retry", screen reader actually still read it as "Sending"

https://msazure.visualstudio.com/CCI/_workitems/edit/15627650

**Comment by compulim** [comment]
I could not repro this on Web Chat loader and PVA, in both scan and browse mode. Tested on Narrator + Edge.

I am asking the tester to understand how they trigger the "send failed" prompt. Maybe the way they trigger the prompt kept the prompt as "sending".

**Comment by compulim** [comment]
The tester closed the original bug as fixed. 🎉
